shared/chat/conversation/messages/wrapper/wrapper-message/index.js

```diff
@@ -141,13 +141,13 @@ class _WrapperMessage extends React.Component<Props & Kb.OverlayParentProps, Sta
 
     return (
       <React.Fragment>
```

Comment:
turns out making this a fragment causes the parent container (made by RN) to lay it out (which is normally fine) but it turns out this uses flexDirection: column-reverse for some reason so it flips the relationship!